README.md

```diff
@@ -55,6 +56,27 @@ Note: grab-site currently **does not work with Python 3.5**; please use Python 3
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+Install with Docker
+---
+Get the pre-built docker container:
```

Comment:
Eh, that depends on your package manager... for me it's `pacman -S docker` (on Arch), but it's `apt-get install` if you're on a debian-based system, `yum` on RHEL/Fedora and I imagine there's something similar for OSX. Or it's pre-installed if you're on CoreOS.